business_requirement/report/report.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <report id="business_requirement_report"
+            model="business.requirement"
+            string="Business Requirement Report"
+            report_type="qweb-pdf"
+            name="business_requirement.br_report"
+            file="business_requirement.br_report"
+    />
+
+    <record id="business_requirement_report" model="ir.actions.report">
+        <field name="paperformat_id"
+               ref="business_requirement.br_report_paperformat_eu"/>
+    </record>
```

Comment:
Why do you create a new paperformat?

Reply:
@Tardo Could you reply this one?

Reply:
removed, used base paperformat instead. Thanks.